### PR TITLE
Be able to create intermediate release

### PR DESCRIPTION
**Pull request by sbrunner**
Named development release or release candidate on minor release.

### CODE REVIEW

File: doc/developer/build_release.rst [hunk 1]
```diff
@@ -152,7 +152,10 @@ Update the version of c2cgeoportal to ``<release>`` in
 ``c2cgeoportal/scaffolds/update/CONST_requirements_windows.txt``.
 
 Verify that the version in the ``setup.py`` is correct
-(as the ``<release>``).
+(as the ``<release>``, required when we create an rc or a dev release).
```
Comment (asaunier) [inline doc/developer/build_release.rst]: Not a big deal, but maybe "RC" instead of "rc" (or even "a release candidate")

Comment (sbrunner) [inline doc/developer/build_release.rst]: Thanks @asaunier fixed :-)

File: doc/developer/build_release.rst [hunk 2]
```diff
@@ -184,10 +188,26 @@ Push your changes:
     git push origin <version>
     git push origin <release>
 
-.. note::
+Rc or dev w.x.y.(rc|dev)z
```
Comment (asaunier) [inline doc/developer/build_release.rst]: Release candidate?

**Comment by asaunier** [comment]
LGTM

**Pull request merged**